explorer/src/services/password.ts

```diff
@@ -0,0 +1,9 @@
+import argon2 from 'argon2'
+
+export function hash(plaintext: string): Promise<string> {
+  return argon2.hash(plaintext)
```

Comment:
might want to use argon2id here: https://github.com/ranisalt/node-argon2/wiki/Options#type

Reply:
@coventry thoughts on this?

Reply:
The default in the argon package is [argon2i](https://github.com/ranisalt/node-argon2/blob/master/test/test.js#L30), I think. Argon2id is preferable if you can afford the resources, because it [demands a lot of memory](https://crypto.stackexchange.com/a/49969), making it harder to parallelize. So I agree with @typescribe, if memory's not an issue. You can set the memory cost with the [`memoryCost` option](https://github.com/ranisalt/node-argon2/blob/master/argon2.js#L38).

Reply:
This endpoint won't be getting hit very often so I'm not very concerned about resource usage unless it's from a malicious source.